environment.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
This file should be removed prior to merger. Added as a convenience for collaborating within our team on these changes.